pyproject.toml

```diff
@@ -1,21 +1,32 @@
 [build-system]
-requires = ["poetry-core>=1.0.0"]
-build-backend = "poetry.core.masonry.api"
+requires = ["hatchling>=1.8.0", "hatch-vcs", "hatch-fancy-pypi-readme"]
+build-backend = "hatchling.build"
 
-[tool.poetry]
+[tool.hatch.version]
+path = "lilio/__init__.py"
+
+[tool.hatch.build.targets.sdist]
+exclude = [
+  "/.github", "/.mypy_cache", "/.pytest_cache", "/.githooks",
+  "sonar-project.properties"
+]
+
+[tool.hatch.build.targets.wheel]
+packages = ["lilio"]
+
+[project]
 name = "lilio"
-version = "0.2.0"
 description = "python package for generating calendars for machine learning timeseries analysis."
+readme = "README.md"
+license = { text = "Apache Software License" }
+requires-python = ">=3.8, <3.11"
 authors = [
-  "Yang Liu <y.liu@esciencecenter.nl>",
-  "Bart Schilperoort <b.schilperoort@esciencecenter.nl>",
-  "Peter Kalverla <b.schilperoort@esciencecenter.nl>",
-  "Jannes van Ingen <jannes.van.ingen@s2s-ai.com>",
-  "Sem Vijverberg <sem.vijverberg@vu.nl>",
+  {name = "Yang Liu", email = "y.liu@esciencecenter.nl"},
+  {name = "Bart Schilperoort", email = "b.schilperoort@esciencecenter.nl"},
+  {name = "Peter Kalverla", email = "b.schilperoort@esciencecenter.nl"},
+  {name = "Jannes van Ingen", email = "jannes.van.ingen@s2s-ai.com"},
+  {name = "Sem Vijverberg", email = "sem.vijverberg@vu.nl"},
```

Comment:
This did not render correctly! Only one author seems to be allowed, despite the field being called "authors"...
[On era5cli](https://pypi.org/project/era5cli/), this also seems to render like a mangled list.

I've decided to put a list of authors in (as a single string), this will render correctly:
```
Name: lilio
Version: 0.2.1
Summary: python package for generating calendars for machine learning timeseries analysis.
Home-page:
Author: Yang Liu, Bart Schilperoort, Peter Kalverla, Jannes van Ingen, Sem Vijverberg
Author-email: y.liu@esciencecenter.nl
License: Apache Software License
Location: c:\users\bartschilperoort\venvs\lilio\lib\site-packages
Requires: matplotlib, netcdf4, numpy, pandas, xarray
```